controllers/shooter.py

```diff
@@ -86,12 +91,20 @@ def tracking(self) -> None:
 
         self.turret.slew_local(angle)
 
-        if self.interpolation_override:
-            self.shooter.motor_speed = self.flywheel_speed
-        else:
-            self.shooter.motor_speed = interpolate(
-                self.distance, self.ranges_lookup, self.flywheel_speed_lookup
-            )
+        if (
+            self.indexer.has_cargo_in_chimney()
+            or self.indexer.has_cargo_in_tunnel()
+            or self.intake.deployed
+        ):
+            self.flywheels_running = True
+
+        if self.flywheels_running:
+            if self.interpolation_override:
+                self.shooter.motor_speed = self.flywheel_speed
+            else:
+                self.shooter.motor_speed = interpolate(
+                    self.distance, self.ranges_lookup, self.flywheel_speed_lookup
+                )
 
```

Comment:
It looks like the shooter component doesn't reset its speed to 0. Oversight?

Reply:
This is now being reset in the component